Docs/Them Lit FYP Information.md

```diff
@@ -26,25 +25,34 @@ The project aims to make exercising fun with VR and Sensors. Body movements will
 # Not Dodgeball VR the game 2020
 
 **Core mechanics:**
-- Two players (A and B).
-- Goal for A is behind B and vice versa.
-- Player's Goal is attached to the player and will move as th player moves, following behind at a set distance.
-- Game starts with each player owning 1 ball
-- Players try to score into their respective goals.
-- Points awarded for goals.
-- Ball can bounce off every surface with physics matching ball material, and wall material.
-- Ball belongs to player who last touched it, and players can own goal.
-- Every ball that is scored causes 2 balls to spawn (TBC).
-- The maximum number of balls allowed in the game is decided by the game host.
-- The duration of a match, as well as the number of matches to count as a game, are also decided by the host.
-- In case of the ball coming to a stop, destroy and spawn back in players hand.
-- If a player's hand remains stationary, the tool will become transparent and balls will not collide with it.
+- Two player game with both players put into a rectangular room at opposite sides.
+- Each player has a Goal which the aim is to score points into the opposing players goal.
```

Comment:
inconsistent capitalisation of "goal", personally i think should be lowercase throughout